saga/seata-saga-spring/src/main/java/org/apache/seata/saga/engine/store/db/DbAndReportTcStateLogStore.java

```diff
@@ -729,11 +729,26 @@ public List<StateInstance> queryStateInstanceListByMachineInstanceId(String stat
     }
 
     @Override
-    public void clearUp() {
+    public void clearUp(StateMachineInstance machineInstance, ProcessContext context) {
```

Comment:
You dont need to add `StateMachineInstance machineInstance` to method signature, use `(StateMachineInstance) context.getVariable(DomainConstants.VAR_NAME_STATEMACHINE_INST)` to get it instead.

It is more natrual to use `ProcessContext` to pass variables.